docs/neo3_guides/token_transfer.md

```diff
@@ -91,9 +91,9 @@ signers and the wallet to the `TransactionBuilder`, so it is not necessary to co
 manually.
 
 ```java
-Account account1 = Account.fromWif("L1WMhxazScMhUrdv34JqQb1HFSQmWeN2Kpc1R9JGKwL7CDNP21uR");
-Account account2 = Account.fromWif("L45BGYyybk91pvwH3Mj1CfDZ11GGQLVPr6qfzpWugeP4WeJZyfki");
-Account account3 = Account.fromWif("L2pN4EbagTuk9Kiib8sjRmMQznxqCVEs1HR8DRaxmnPicjg9FdNc");
+Account account1 = Account.fromWIF("L1WMhxazScMhUrdv34JqQb1HFSQmWeN2Kpc1R9JGKwL7CDNP21uR");
```

Comment:
These WIF are still compatible with neow3j? After we changed the default `AddressVersion`?

Reply:
They actually are, but while you commented, I was just updating some addresses in the docs and adjusted accounts that are created using the method `fromWIF()` to the ones used in `neo3-privatenet-docker`.